build.gradle

```diff
@@ -142,10 +142,10 @@ repositories {
         name 'amadornes'
         url 'http://maven.amadornes.com/'
     }
-    maven { // Tesla
-        name 'epoxide'
-        url 'http://maven.epoxide.org'
-    }
+//    maven { // Tesla
+//        name 'epoxide'
+//        url 'http://maven.epoxide.org'
+//    }

```

Comment:
Please stop using the epxode maven. Tesla is available on the MMD maven now.

Reply:
well... that's commented out :)